static/js/components/CardGrid.js

```diff
@@ -1,7 +1,7 @@
 import React, { Component } from "react";
 import CardComponent from "./CardComponent"
 
-require('../../css/CardComponent.scss');
+// require('../../css/CardComponent.scss');
```

Comment:
Same with this line, can it just be deleted?

Reply:
Yes, will fix